traffic_control/clients/python/trafficops/tosession.py

```diff
@@ -1522,6 +1522,38 @@ def delete_multiple_servers_capabilities(self, data=None):
 		:raises: Union[LoginError, OperationError]
 		"""
 
+	#
+	# Server_server_capabilities
+	#
+	@api_request('get', 'server_server_capabilities', ('3.0', '4.0', '4.1', '5.0'))
+	def get_server_server_capabilities(self, query_params=none):
+		"""
+		Gets all associations of Server Capabilities to cache servers.
+		:ref:`to-api-server_server_capabilities`
+		:rtype: Tuple[Dict[str, Any], requests.Response]
+		:raises: Union[LoginError, OperationError]
+		"""
+
+
+	@api_request('post', 'server_server_capabilities', ('3.0', '4.0', '4.1', '5.0'))
+	def associate_server_capability_to_server(self, data=None):
+		"""
+		Associate server capability to server.
```

Comment:
missing annotation for data param 
`:param data: The parameter data to .......... `
:type data: Dict[str, Any]

Reply:
added annotation for data param